src/en/help-azure.md

```diff
@@ -90,24 +97,21 @@ data:    Free Trial  f717c8c1-8e5e-4d38-be7f-ed1e1c879e18  true     Enabled
 info:    account list command OK
 ```
 
-The subscription ID can now be stored in a local environment variable, by
-entering:
+In this sample,
 
 ```bash
 SUB_ID=f717c8c1-8e5e-4d38-be7f-ed1e1c879e18
 ```
```

Comment:
We do not show output to the other commands. Why show it for just this command? Also, to be consistent with the below stuff, we shoulds say "Note the application ID, the **SUB_ID**. It will look similar to:...".